src/renderer/missionWindow/MissionWindow.tsx

```diff
@@ -4,15 +4,18 @@ import { Event, ipcRenderer } from 'electron';
 import React, { Component, ReactNode } from 'react';
 import * as Slider from 'rc-slider';
 
+import { BoundingBoxBounds, ThemeProps } from '../../types/componentStyle';
+
 import { JobType } from '../../static/index';
 
-import { ThemeProps } from '../../types/componentStyle';
```

Comment:
move line 7 back here

Reply:
Where to exactly? the import { JobType } goes to line 7 and the import { BoundingBoxBounds, ThemeProps } goes where?

Reply:
See the image below to see what I mean
![image](https://user-images.githubusercontent.com/13097797/75592854-3f9acf00-5a38-11ea-9752-c979fec30e72.png)